aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecCommentSpan.kt

```diff
@@ -0,0 +1,46 @@
+package org.wordpress.aztec.spans
+
+import android.content.Context
+import android.graphics.Paint
+import android.graphics.Rect
+import android.graphics.drawable.Drawable
+import android.text.style.ImageSpan
+
+class AztecCommentSpan(val comment: AztecCommentSpan.Comment, val context: Context, drawable: Drawable) : ImageSpan(drawable) {
+    companion object {
+        private val HTML_MORE: String = "more"
+        private val HTML_PAGE: String = "nextpage"
+    }
+
+    enum class Comment constructor(val html: String) {
+        MORE(HTML_MORE),
+        PAGE(HTML_PAGE)
+    }
+
+    override fun getSize(paint: Paint?, text: CharSequence?, start: Int, end: Int, metrics: Paint.FontMetricsInt?): Int {
+        val drawable = drawable
+        val bounds = getBounds(drawable)
+
+        if (metrics != null) {
+            metrics.ascent = -bounds.bottom
+            metrics.descent = 0
+
+            metrics.top = metrics.ascent
+            metrics.bottom = 0
+        }
+
+        return bounds.right
+    }
+
+    private fun getBounds(drawable: Drawable): Rect {
+        if (drawable.intrinsicWidth === 0) {
+            return Rect(0, 0, drawable.intrinsicWidth, drawable.intrinsicHeight)
```

Comment:
I think performance wise it might be a better idea to reuse Rect. Declare it in a class and then just call something like this:
```
 rect.set(0, 0, drawable.intrinsicWidth, drawable.intrinsicHeight)
 return rect
```